From 85970f70b827f8dfe826e511de3d89abef18d6f2 Mon Sep 17 00:00:00 2001
From: Ray Chiang <rchiang@cloudera.com>
Date: Tue, 24 Nov 2015 10:11:34 -0800
Subject: [PATCH 0980/1023] CLOUDERA-BUILD. MR1. Update Apache
 commons-collection version to 3.2.2.

Change-Id: I92d8a523d26b5b68e1e5dd5def0a779a99363e31
(cherry picked from commit db1c7b9c69ce8cdbbf7ce9135e4d240e468b4a89)
---
 hadoop-mapreduce1-project/ivy/libraries.properties |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-mapreduce1-project/ivy/libraries.properties b/hadoop-mapreduce1-project/ivy/libraries.properties
index 2dd5eae..b29744f 100644
--- a/hadoop-mapreduce1-project/ivy/libraries.properties
+++ b/hadoop-mapreduce1-project/ivy/libraries.properties
@@ -26,7 +26,7 @@ checkstyle.version=4.2
 
 commons-cli.version=1.2
 commons-codec.version=1.4
-commons-collections.version=3.1
+commons-collections.version=3.2.2
 commons-httpclient.version=3.1
 commons-lang.version=2.5
 commons-logging.version=1.1.3
-- 
1.7.9.5

